openpype/hosts/houdini/plugins/create/create_review.py

```diff
@@ -66,9 +62,96 @@ def create(self, subset_name, instance_data, pre_create_data):
 
         instance_node.setParms(parms)
 
-        # Lock any parameters in this list
-        to_lock = [
-            "family",
-            "id"
-        ]
+        to_lock = ["id", "family"]
+
         self.lock_parameters(instance_node, to_lock)
+
+    def get_pre_create_attr_defs(self):
+        attrs = super().get_pre_create_attr_defs()
+        image_format_enum = [
+            {
+                "value": "png",
+                "label": ".png"
+            },
+            {
+                "value": "tif",
+                "label": ".tif"
+            },
+            {
+                "value": "sgi",
+                "label": ".sgi"
+            },
+            {
+                "value": "pic.gz",
+                "label": ".pic.gz"
+            },
+            {
+                "value": "rat",
+                "label": ".rat"
+            },
+            {
+                "value": "jpg",
+                "label": ".jpg"
+            },
+            {
+                "value": "cin",
+                "label": ".cin"
+            },
+            {
+                "value": "rta",
+                "label": ".rta"
+            },
+            {
+                "value": "rat",
+                "label": ".rat"
+            },
+            {
+                "value": "bmp",
+                "label": ".bmp"
+            },
+            {
+                "value": "tga",
+                "label": ".tga"
+            },
+            {
+                "value": "rad",
+                "label": ".rad"
+            },
+            {
+                "value": "exr",
+                "label": ".exr"
+            },
+            {
+                "value": "pic",
+                "label": ".pic"
+            }
+        ]
```

Comment:
Ouch, so big.

Doesn't it also work like:
```python
image_format_enum= {
    "pic": ".pic",
    "exr": ".exr",
    "tga": ".tga",
    "bmp": ".bmp"  
}
```

Or even just:
```python
{".{}".format(ext): ext for ext in ["pic", "exr", "rad", "tga", "bmp", "rat"]}
```
Even though that might be pushing it a bit too far.

Reply:
Not quite sure, I take reference from https://github.com/ynput/OpenPype/pull/4588 for the same hosts
https://github.com/ynput/OpenPype/blob/af471f6bb3ed236e0b80cde34abe2d00493baed0/openpype/hosts/houdini/plugins/create/create_bgeo.py#L54-L79

Reply:
Fixed it on my end. Thanks!